client/js/socket-events/configuration.js

```diff
@@ -32,3 +32,8 @@ socket.once("configuration", function (data) {
 		});
 	}
 });
+
+socket.on("configuration:reload", function () {
```

Comment:
I'm not sure if this should be scoped to `configuration` just because it's only used for it, as the actual content isn't at all config related. Personal preference atm, I guess. Seeing `configuration:reload` in ClientManager feels a little awkward.